engine/src/test/java/com/datatorrent/stram/engine/StreamingContainerTest.java

```diff
@@ -94,4 +121,33 @@ public void emitTuples()
     private static final Logger logger = LoggerFactory.getLogger(CommitAwareOperator.class);
   }
 
+  private static class CommitAwareInputOperator extends CommitAwareOperator
+  {
+    public final transient DefaultOutputPort<String> output = new DefaultOutputPort<String>();
+  }
+
+  private static class CommitAwareProcessOperator extends CommitAwareOperator
+  {
+    private static final Logger logger = LoggerFactory.getLogger(CommitAwareOperator.class);
+    public static ArrayList<Long> committedWindowIds;
```

Comment:
If CommitAwareProcessOperator were a non static inner class, it could not have static field logger.